optimize/client/src/modules/components/ReportRenderer/HyperReportRenderer.js

```diff
@@ -9,13 +9,29 @@
 import React from 'react';
 import update from 'immutability-helper';
 
-import {formatters, getReportResult} from 'services';
+import {formatters, getReportResult, processResult as processSingleReportResult} from 'services';
 
-import CombinedReportRenderer from './CombinedReportRenderer';
+import {Table, Chart} from './visualizations';
 import ProcessReportRenderer from './ProcessReportRenderer';
+import {getFormatter} from './service';
 
 const {formatReportResult} = formatters;
 
+const getComponent = (visualization) => {
```

Comment:
I see this code was kinda moved here from `CombinedReportRenderer` does this mean that we were lacking this functionality in here before or what's the reason of moving these things in here?